src/django-app/README.md

```diff
@@ -0,0 +1,65 @@
+# Setup Instructions
+
+## Step 1: Create a Postgres DB
+- Install PostgreSQL (if not already installed)
+- On the terminal, run:
+```bash
+psql
```

Comment:
When first running this you will have to run the command `psql -U postgres` to login as the postgres user so you can set things up. I don't know if you want to go into installing postgres too much, but maybe just make this change to make installation smoother for first timers:

```suggestion
psql -U postgres
```